packages/sentry-client/src/utils/prettyTime.ts

```diff
@@ -0,0 +1,42 @@
+export function pretty(seconds: number, scale: number = 1): string {
```

Comment:
Weeks is probably the largest timescale we need to deal with (`uptime` util does the same.) And this is a relatively small file which I prefer to having to use all of moment.js or similar.